packages/web-components/src/components/masthead/defs.ts

```diff
@@ -45,4 +45,17 @@ export enum MEGAMENU_RIGHT_NAVIGATION_STYLE_SCHEME {
    * For left (highlighted) section layout.
    */
   LEFT_SECTION = 'left-section',
+
+  /**
+   * For tabbed megamenus.
+   */
+  TAB = 'tab',
+}
+
+/**
+ * The layout options for rendering a megamenu
+ */
+export enum MEGAMENU_LAYOUT_SCHEME {
+  TAB = 'tab',
```

Comment:
Question: I like the addition of this `defs` file for defining the different mega menu layouts that are available. For the Tab mega menu layout with the secondary level heading, are we looking to expose another tab layout option, or is the rendering of these secondary levels (see data schema - https://github.ibm.com/webstandards/ibm-dotcom-translations/blob/master/translations/masthead-footer/v2/usen.json#L2074-L2112) will be determined in the megamenu right navigation return? This is more of future phase question.

Reply:
You're talking about how the individual tabs' layouts will be variable in a future phase, correct? I envision the tabs' declaring their own layouts in the data schema similarly to how L0 items do so in the latest schema version. Then we can render different versions of the right nav based on that information.

Reply:
@jkaeser Makes sense. Can you submit a JIRA ticket for this additive field so that we can track work for the future phase of the L0 megamenu update and eventually open a PR against the existing translation repo?

Reply:
Created https://jsw.ibm.com/browse/ADCMS-2561 for that task.